main/src/ca/uwaterloo/flix/util/collection/MultiMap.scala

```diff
@@ -18,32 +18,32 @@ object MultiMap {
 /**
   * Represents a map from keys of type `K` to sets of values of type `V`.
   */
-case class MultiMap[K, V](m: Map[K, Set[V]]) {
```

Comment:
Wait. You should not change MultiMap-- there are other uses of MultiMap where the values must be a set!

Reply:
Oops. Fixed.